src/android/usb_host/android_uvc.cpp

```diff
@@ -938,19 +938,27 @@ void stream_thread(usbhost_uvc_stream_context *strctx) {
         }
     });
     LOG_DEBUG("Transfer thread started for endpoint address: " << strctx->endpoint);
+    bool disconnect = false;
     do {
         auto i = strctx->stream->stream_if->interface;
         uint32_t transferred = 0;
         auto sts = messenger->bulk_transfer(read_ep, strctx->stream->outbuf, LIBUVC_XFER_BUF_SIZE, transferred, 1000);
```

Comment:
Why do you use hard-coded timeouts ? (1000, 50)

Reply:
will create a define, it is not accessible to the app level